include/vinecopulib/misc/tools_eigen.hpp

```diff
@@ -21,7 +21,11 @@ template<typename T>
 Eigen::MatrixXd unaryExpr_or_nan(const Eigen::MatrixXd &x, const T &func)
 {
     return x.unaryExpr([&func](const double &y) {
-        return tools_stl::unaryFunc_or_nan(func, y);
```

Comment:
Why is your thing better?

Reply:
(and same question for the binary)

Reply:
tools_stl header no leader needs to be included in tools_eigen. The stuff in tools_stl was only used within the tools_eigen header, so why not have the code in tools_eigen directly.

Reply:
OK